Client/Application/LeanplumIntegration.swift

```diff
@@ -114,11 +115,14 @@ class LeanplumIntegration {
             profile?.prefs.setBool(!canInstallFocus(), forKey: PrefsKeys.HasFocusInstalled)
         }
 
+        let appVersion = Float(Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String)
```

Comment:
This won't work. `CFBundleShortVersionString` is a string, which can also be `8.1.1` (as it is currenty) or even `9.0b1` - neither are numbers.